web/package-lock.json

```diff
@@ -38,7 +38,7 @@
                 "ts-loader": "^9.5.1",
                 "tsx": "^4.16.2",
                 "typescript": "^5.5.3",
-                "typescript-eslint": "^7.15.0",
+                "typescript-eslint": "^8.0.0-alpha.39",
```

Comment:
Same (especially) here.

Reply:
Same here; first version to support eslint9

They both seem pretty good though, and typescript-eslint got some upgraded rules from it